### PR TITLE
First commit to add user agent header.

### PR DESCRIPTION
**Pull request by matt1883**

### CODE REVIEW

File: azure/datalake/store/lib.py [hunk 1]
```diff
@@ -207,6 +207,10 @@ def __init__(self, store_name=default_store, token=None,
         self.token = token
         self.head = {'Authorization': 'Bearer ' + token['access']}
         self.url = 'https://%s.%s/webhdfs/v1/' % (store_name, url_suffix)
+        self.user_agent = "python/{} ({}) {}/{} Azure-Data-Lake-Store-SDK-For-Python".format(platform.python_version(),
```
Comment (begoldsm) [inline azure/datalake/store/lib.py]: @matt1883  you need to include the platform module if you are going to use

Comment (matt1883) [inline azure/datalake/store/lib.py]: done

**Comment by begoldsm** [comment]
@matt1883 please include a sample of what this looks like by running a scenario with debug logging and paste into the comments.

**Comment by matt1883** [comment]
User-Agent: python/3.5.2 (Windows-10-10.0.14393-SP0) azure.datalake.store.lib/0.0.1 Azure-Data-Lake-Store-SDK-For-Python

**Pull request merged**